### DISCUSSION TITLE
Public/Private CWMS locations

### DISCUSSION THREAD

**Discussion by JeremyDKellett** [USACE/cwms-data-api#864] [Ideas]
Is there a roadmap for adding a public/private flag on CWMS Locations/sub locations? I.e. authenticated vs public? I could see situations in CWMS_GAGE (and others) where the some locations of a gage wouldn't be applicable to public viewers. Anybody else think this be useful?

**Comment by MikeNeilson** [comment]
Not yet, but it will likely be more complex than a simple flag given all the data is going to be under a single database.
